schema/data.xsd

```diff
@@ -1,3 +1,117 @@
+                <xs:element name="uint144" type="variable"/>
+                <xs:element name="uint152" type="variable"/>
+                <xs:element name="uint160" type="variable"/>
+                <xs:element name="uint168" type="variable"/>
+                <xs:element name="uint176" type="variable"/>
+                <xs:element name="uint184" type="variable"/>
+                <xs:element name="uint192" type="variable"/>
+                <xs:element name="uint200" type="variable"/>
+                <xs:element name="uint208" type="variable"/>
+                <xs:element name="uint216" type="variable"/>
+                <xs:element name="uint224" type="variable"/>
+                <xs:element name="uint232" type="variable"/>
+                <xs:element name="uint240" type="variable"/>
+                <xs:element name="uint248" type="variable"/>
+                <xs:element name="uint256" type="variable"/>
+                <xs:element name="int8" type="variable"/>
+                <xs:element name="int16" type="variable"/>
+                <xs:element name="int24" type="variable"/>
+                <xs:element name="int32" type="variable"/>
+                <xs:element name="int40" type="variable"/>
+                <xs:element name="int48" type="variable"/>
+                <xs:element name="int56" type="variable"/>
+                <xs:element name="int64" type="variable"/>
+                <xs:element name="int72" type="variable"/>
+                <xs:element name="int80" type="variable"/>
+                <xs:element name="int88" type="variable"/>
+                <xs:element name="int96" type="variable"/>
+                <xs:element name="int104" type="variable"/>
+                <xs:element name="int112" type="variable"/>
+                <xs:element name="int120" type="variable"/>
+                <xs:element name="int128" type="variable"/>
+                <xs:element name="int136" type="variable"/>
+                <xs:element name="int144" type="variable"/>
+                <xs:element name="int152" type="variable"/>
+                <xs:element name="int160" type="variable"/>
+                <xs:element name="int168" type="variable"/>
+                <xs:element name="int176" type="variable"/>
+                <xs:element name="int184" type="variable"/>
+                <xs:element name="int192" type="variable"/>
+                <xs:element name="int200" type="variable"/>
+                <xs:element name="int208" type="variable"/>
+                <xs:element name="int216" type="variable"/>
+                <xs:element name="int224" type="variable"/>
+                <xs:element name="int232" type="variable"/>
+                <xs:element name="int240" type="variable"/>
+                <xs:element name="int248" type="variable"/>
+                <xs:element name="int256" type="variable"/>
+                <xs:element name="string" type="variable"/>
+                <xs:element name="bytes" type="variable"/>
```

Comment:
Still need bytes1, even though it is the same as byte

Reply:
> @colourful-land might be good to add arrays to this Schema version as well?

No let's avoid adding it only to immediately change it in the next schema. Reason: ASN.1 compatibility desirable.